<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>etages_rayons</class>
 <widget class="QDialog" name="etages_rayons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Etagers &amp; Rayons</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/logo.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Consulter Etagers &amp; Rayons</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>521</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox {
border: 2px solid black;
border-radius: 4px;
}</string>
   </property>
   <property name="title">
    <string>Ajouter nouveau Etagers :</string>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre Rayons :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_nbrRayon">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_ajouter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#bt_ajouter{
color:black;
background-color:#a0a0a0;
border-width:0px;
border-radius:4px;
}
#bt_ajouter:hover{

background-color:#27a9e3;

}</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabe_etagers">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>156</number>
    </attribute>
    <column>
     <property name="text">
      <string>ID ETAGER</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOMBRE RAYONS</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Capacite de rayon :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_capRayon">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>651</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_2 {
border: 2px solid black;
border-radius: 4px;
}</string>
   </property>
   <property name="title">
    <string>Consulter Rayons :</string>
   </property>
   <widget class="QTableWidget" name="table_rayon">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>511</width>
      <height>271</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>122</number>
    </attribute>
    <column>
     <property name="text">
      <string>ID_RAYON</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID ETAGER</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CAPACITE RAYON</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID Etagere :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_etager">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>TOUS</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
